0.9 JS Fundamentals - DOM/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Submit Blog</title>
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
```

Comment:
is it possible to move this script to a JS file instead of writing it in HTML?

Reply:
Yes I will move this in separate file